Topic3/src/org/maugiam/tdd/test/AmountConverterTest.java

```diff
@@ -0,0 +1,35 @@
+package org.maugiam.tdd.test;
+
+import junit.framework.TestCase;
+
+import org.junit.Test;
+import org.maugiam.tdd.main.AmountConverter;
+
+public class AmountConverterTest extends TestCase {
+
+	AmountConverter converter;
+
+	public void setUp() {
+		converter = new AmountConverter();
+	}
+
+	@Test
+	public void testCreateConverter() {
+		assertNotNull(converter);
+	}
+
+	@Test
+	public void testSimpleAmountConver() {
+		assertEquals("One dollar", converter.convert(1.00));
+	}
+
+	@Test
+	public void testComplexAmountConver() {
+		assertEquals("Two thousand five hundred twenty-three and 04/100 dollars", converter.convert(2523.04));
+		System.out.println(converter.convert(1217.0));

```

Comment:
Warning: Why is this line here?. I guess it was used just to see if the converter was working fine, but if you use TDD the converter was not implemented at the moment the test was created. Was the test created first :) ?